dev_docs/contributing/how_we_use_github.mdx

```diff
@@ -37,6 +37,26 @@ Pull requests are made into the master branch and then backported when it is saf
 - Resolve merge conflicts by rebasing the target branch over your feature branch, and force-pushing (see below for instructions).
 - When merging, we’ll squash your commits into a single commit.
 
+### Commit using your `@elastic.co` email address
+
+In order to assist with developer tooling we ask that all Elastic engineers use their `@elastic.co` email address when committing to the Kibana repo. We have implemented a CI check that validates any PR opened by a member of the `@elastic` organization has at least one commit that is attributed to an `@elastic.co` email address. If you have a PR that is failing because of this check you can fix your PR by following these steps:
+
+ 1. Ensure that you don't have any staged changes
+ 1. Checkout the branch for your PR
+ 1. Update the git config for your current repository to commit with your `@elastic.co` email:
```

Comment:
Should we add a step for making sure the email is added to the GitHub account profile?  I expect everyone's already aware,  so not worried if we want to skip this step.  Feel free to resolve.

Reply:
Probably a good idea, to ensure. Thanks